migrations/20240321211121_block.up.sql

```diff
@@ -0,0 +1,7 @@
+-- Add up migration script here
+
```

Comment:
Do we want to create a dedicated schema rather than use public? Might make a change to another db (e.g. sqlite to postgres) easier if we were to ever support that.